tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "jsxImportSource": "@emotion/react"
+    "jsxImportSource": "@emotion/react",
+    "incremental": true
```

Comment:
This change with `incremental` was deliberate?

Reply:
yep we need that with the latest version of nextjs